### ISSUE TITLE
Upgrade to fp-ts 2

### ISSUE THREAD

**Issue by steida** [gcanti/money-ts#7]
Do you plan it or should userland send a PR? I know the company which maybe will desperately need it.

**Comment by gcanti** [comment]
Yes I do (as soon as I understand how to use the new `bigint` type [introduced in TypeScript 3.2](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html))

**Comment by EricCrosson** [comment]
I too am investigating alternative libraries for currency manipulation. I tried money-ts some months ago and found the incompatibility with fp-ts 2 insurmountable. Here we are on the brink of fp-ts 3!

I haven't popped the hood to look at the internals yet but would value the ability to use this library highly. It sounds like there's some modernization required, in terms of `bigint` as well as the fp-ts peer dependency. What would be the best way to help?

**Comment by ryanleecode** [comment]
@EricCrosson did you ever find a proper substitute for this library?

**Comment by EricCrosson** [comment]
No @ryanleecode I have not. I did "start" a tiny (unpublished) version locally for a project but it is still in a rough state.
